js/ui/scheduler/appointmentPopup/popup.js

```diff
@@ -25,7 +25,6 @@ const POPUP_CONFIG = {
     maxHeight: '100%',
     showCloseButton: false,
     showTitle: false,
-    preventScrollEvents: false,
```

Comment:
Unfortunately, we cannot stop using this deprecated option.
This option is needed for the zoom & swipe events on mobile devices.
Without this option, these actions will not work correctly.

We can ask @EugeniyKiyashko for a more detailed answer as to why we should continue using this option :)

For disabling console warnings, Eugeniy suggests using the ```_ignorePreventScrollEventsDeprecation: true```
```javascript
const POPUP_CONFIG = {
  // other popup options.
  preventScrollEvents: false,
  _ignorePreventScrollEventsDeprecation: true,
  // other popup options.
};
```